packages/react/src/ErrorBoundary.ko.mdx

```diff
@@ -48,6 +48,38 @@ const ErrorAfter4s = () => {
 
 ![Example banner](/gif/errorboundary-example.gif)
 
+:::tip
+
+## `<ErrorBoundary/>` fallback props을 prop drilling 없이 사용하기
```

Comment:
```suggestion
## `<ErrorBoundary/>` fallback props을 props drilling 없이 사용하기
```

how about like this?

Reply:
Thanks, but I think `prop drilling` is right (https://react.dev/learn/passing-data-deeply-with-context#the-problem-with-passing-props)